ModulBot/Database/PostgresRepositories/PostgresUserRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ModulBot.Models;
+
+namespace ModulBot.Database.PostgresRepositories
+{
+    public class PostgresUserRepository : IRepository<User>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix this implementation of 'IDisposable' to conform to the dispose pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3881)